spec/schemas/_common.aggregations.yaml

```diff
@@ -1440,7 +1442,7 @@ components:
             - properties
             - type
     AggregationContainer:
-      allOf:
+      allOf: 
```

Comment:
Rationale behind this change as explained in this [commit](https://github.com/opensearch-project/opensearch-api-specification/pull/818/commits/ba6ffee6afee5f5dbd4a61fff1abac222780f33d)

![Screenshot 2025-02-17 at 14 40 28](https://github.com/user-attachments/assets/c34d2f43-6454-43a7-a704-8945563f8478)